rules/game_process.tex

```diff
@@ -517,6 +517,16 @@ \subsection{Mercy Rule}
 A game will conclude once the game score shows a goal difference of 10.
 Ending the game is mandatory once a goal difference of 10 is reached.
 
+\subsection{Drop-Ball Rule}
+\label{sec:drop_ball_rule}
+
+In the event that there has been a referee error or a pause in game, and if a goal has not been scored by either team, the drop-ball rule shall apply. When resuming the game, the process should be the same as a normal start, however, none of the teams will be given kickoff, and no team would be allowed in the center circle until the \texttt{playing} state as there is no longer time allotted for kickoff. The only main example currently where the drop-ball rule would be required is when a referee timeout has been called with no goal being scored.
+
```

Comment:
This section needs to clarify a few things. 
- I presume a referee error needs to be acknowledged by the referee.
- What pauses in the game can occur? If it is just a referee timeout, then we should probably state that.
- What does "if a goal has not been scored by either team" mean? Does this mean at any point previously, or immediately beforehand, or within some particular time window?
- Does a drop ball mean that ball will always be placed in the center of the field as for a standard kickoff? This should be stated. It also seems like quite a severe reset in case of a referee error.
- If the ball is always placed in the center, where do robots start from? Is it from the sidelines? If so, does this mean that in case of any referee error or referee timeout is called, the robots stop playing and are placed on the sideline?

Reply:
I thought of this as the last kickoff that was executed, regardless of whether it was the first half or after a timeout. In the case of a goal was previously scored, and that initiated a kickoff, I would say the conceding team would have kickoff.